tests/templates/kuttl/smoke/helm-bitnami-minio-values.yaml.j2

```diff
@@ -1,4 +1,32 @@
 ---
+mode: standalone
+disableWebUI: false
+extraEnvVars:
+  - name: BITNAMI_DEBUG
+    value: "true"
+  - name: MINIO_LOG_LEVEL
+    value: DEBUG
+
+#defaultBuckets: hive
+
+provisioning:
+  enabled: true
+  buckets:
+    - name: hive
```

Comment:
For some reason I cannot make suggestions here, but please replace `hive` with `trino`